util/log/log.go

```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const timeFormat = "1/2 15:04:05.000" // m/d h:m:s.nanos
```

Comment:
I left it out deliberately. The year seems the most redundant thing for a human running a local cluster.